{"level":"error","message":"Uncaught internal server error. { MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)\n  name: 'MongoError',\n  message: 'server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}' } MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)","timestamp":"2016-08-09T19:03:59.533Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)\n  name: 'MongoError',\n  message: 'server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}' } MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)","timestamp":"2016-08-09T19:03:59.538Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)\n  name: 'MongoError',\n  message: 'server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}' } MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)","timestamp":"2016-08-09T19:03:59.541Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)\n  name: 'MongoError',\n  message: 'server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}' } MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)","timestamp":"2016-08-09T19:03:59.543Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)\n  name: 'MongoError',\n  message: 'server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}' } MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)","timestamp":"2016-08-09T19:03:59.545Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)\n  name: 'MongoError',\n  message: 'server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}' } MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)","timestamp":"2016-08-09T19:03:59.546Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)\n  name: 'MongoError',\n  message: 'server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}' } MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)","timestamp":"2016-08-09T19:04:00.034Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)\n  name: 'MongoError',\n  message: 'server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}' } MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)","timestamp":"2016-08-09T19:04:00.058Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)\n  name: 'MongoError',\n  message: 'server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}' } MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)","timestamp":"2016-08-09T19:04:00.060Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)\n  name: 'MongoError',\n  message: 'server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}' } MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)","timestamp":"2016-08-09T19:04:00.062Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)\n  name: 'MongoError',\n  message: 'server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}' } MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)","timestamp":"2016-08-09T19:04:00.063Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)\n  name: 'MongoError',\n  message: 'server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}' } MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)","timestamp":"2016-08-09T19:04:00.064Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)\n  name: 'MongoError',\n  message: 'server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}' } MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)","timestamp":"2016-08-09T19:04:01.285Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)\n  name: 'MongoError',\n  message: 'server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}' } MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)","timestamp":"2016-08-09T19:04:01.297Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)\n  name: 'MongoError',\n  message: 'server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}' } MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)","timestamp":"2016-08-09T19:04:01.297Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)\n  name: 'MongoError',\n  message: 'server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}' } MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)","timestamp":"2016-08-09T19:04:01.299Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)\n  name: 'MongoError',\n  message: 'server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}' } MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)","timestamp":"2016-08-09T19:04:01.302Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)\n  name: 'MongoError',\n  message: 'server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}' } MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)","timestamp":"2016-08-09T19:04:01.309Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)\n  name: 'MongoError',\n  message: 'server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}' } MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)","timestamp":"2016-08-09T19:04:02.075Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)\n  name: 'MongoError',\n  message: 'server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}' } MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)","timestamp":"2016-08-09T19:04:02.077Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)\n  name: 'MongoError',\n  message: 'server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}' } MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)","timestamp":"2016-08-09T19:04:02.082Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)\n  name: 'MongoError',\n  message: 'server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}' } MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)","timestamp":"2016-08-09T19:04:02.085Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)\n  name: 'MongoError',\n  message: 'server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}' } MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)","timestamp":"2016-08-09T19:04:02.086Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)\n  name: 'MongoError',\n  message: 'server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}' } MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)","timestamp":"2016-08-09T19:04:02.088Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)\n  name: 'MongoError',\n  message: 'server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}' } MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)","timestamp":"2016-08-09T19:04:02.898Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)\n  name: 'MongoError',\n  message: 'server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}' } MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)","timestamp":"2016-08-09T19:04:02.904Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)\n  name: 'MongoError',\n  message: 'server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}' } MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)","timestamp":"2016-08-09T19:04:02.906Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)\n  name: 'MongoError',\n  message: 'server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}' } MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)","timestamp":"2016-08-09T19:04:03.901Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)\n  name: 'MongoError',\n  message: 'server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}' } MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)","timestamp":"2016-08-09T19:04:03.913Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)\n  name: 'MongoError',\n  message: 'server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}' } MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)","timestamp":"2016-08-09T19:04:03.917Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)\n  name: 'MongoError',\n  message: 'server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}' } MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)","timestamp":"2016-08-09T19:04:09.260Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)\n  name: 'MongoError',\n  message: 'server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}' } MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)","timestamp":"2016-08-09T19:04:09.274Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)\n  name: 'MongoError',\n  message: 'server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}' } MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)","timestamp":"2016-08-09T19:04:09.470Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)\n  name: 'MongoError',\n  message: 'server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}' } MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)","timestamp":"2016-08-09T19:04:09.564Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)\n  name: 'MongoError',\n  message: 'server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}' } MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)","timestamp":"2016-08-09T19:04:09.953Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)\n  name: 'MongoError',\n  message: 'server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}' } MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)","timestamp":"2016-08-09T19:04:10.088Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)\n  name: 'MongoError',\n  message: 'server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}' } MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)","timestamp":"2016-08-09T19:04:10.279Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)\n  name: 'MongoError',\n  message: 'server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}' } MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)","timestamp":"2016-08-09T19:04:11.047Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)\n  name: 'MongoError',\n  message: 'server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}' } MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)","timestamp":"2016-08-09T19:04:11.113Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)\n  name: 'MongoError',\n  message: 'server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}' } MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)","timestamp":"2016-08-09T19:04:12.165Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)\n  name: 'MongoError',\n  message: 'server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}' } MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)","timestamp":"2016-08-09T19:05:17.804Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)\n  name: 'MongoError',\n  message: 'server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}' } MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)","timestamp":"2016-08-09T19:05:17.810Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)\n  name: 'MongoError',\n  message: 'server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}' } MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)","timestamp":"2016-08-09T19:05:17.850Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)\n  name: 'MongoError',\n  message: 'server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}' } MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)","timestamp":"2016-08-09T19:05:17.934Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)\n  name: 'MongoError',\n  message: 'server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}' } MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)","timestamp":"2016-08-09T19:05:17.997Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)\n  name: 'MongoError',\n  message: 'server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}' } MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)","timestamp":"2016-08-09T19:05:18.020Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)\n  name: 'MongoError',\n  message: 'server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}' } MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)","timestamp":"2016-08-09T19:05:18.092Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)\n  name: 'MongoError',\n  message: 'server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}' } MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)","timestamp":"2016-08-09T19:05:18.284Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)\n  name: 'MongoError',\n  message: 'server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}' } MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)","timestamp":"2016-08-09T19:05:18.550Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)\n  name: 'MongoError',\n  message: 'server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}' } MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)","timestamp":"2016-08-09T19:05:18.570Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)\n  name: 'MongoError',\n  message: 'server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}' } MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)","timestamp":"2016-08-09T19:05:19.052Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)\n  name: 'MongoError',\n  message: 'server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}' } MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)","timestamp":"2016-08-09T19:05:19.298Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)\n  name: 'MongoError',\n  message: 'server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}' } MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)","timestamp":"2016-08-09T19:05:19.484Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)\n  name: 'MongoError',\n  message: 'server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}' } MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)","timestamp":"2016-08-09T19:05:19.678Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)\n  name: 'MongoError',\n  message: 'server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}' } MongoError: server ds139665-a.mlab.com:39665 received an error {\"name\":\"MongoError\",\"message\":\"read ETIMEDOUT\"}\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/topologies/server.js:213:40)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at .<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/pool.js:119:12)\n    at g (events.js:286:16)\n    at emitTwo (events.js:106:13)\n    at emit (events.js:191:7)\n    at Socket.<anonymous> (/Users/raineroviir/ParseServer/node_modules/mongodb-core/lib/connection/connection.js:151:49)\n    at Socket.g (events.js:286:16)","timestamp":"2016-08-09T19:05:19.853Z"}
